hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/systemcontracts/hts/grantapproval/ClassicGrantApprovalCall.java

```diff
@@ -27,6 +27,12 @@
  * Implements the token redirect {@code approve()} call of the HTS system contract
  */
 public class ClassicGrantApprovalCall extends AbstractGrantApprovalCall {
+
+    // Approval(address indexed owner, address indexed spender, uint256 value)
+    // Approval(address indexed owner, address indexed approved, uint256 indexed tokenId)
+    protected static final Bytes APPROVAL_EVENT =
```

Comment:
Can we have a better explanation of these bytes and how they are derived.  It will be difficult for future maintainers to understand where these bytes come from